_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+---
+title:Recommendations Report
+date: 2019-08-02
+authors:
+  - Service Innovation Lab
```

Comment:
the build if failing because there is no author named "Service Innovation Lab".
It'll need to be some in the teams list:
https://github.com/ServiceInnovationLab/serviceinnovationlab.github.io/tree/dev/_authors